.github/workflows/lint.yaml

```diff
@@ -40,6 +40,12 @@ jobs:
               - 'Cargo.lock'
               - 'build.assets/versions.mk'
               - 'Makefile'
+            has_proto:
```

Comment:
Do we want to do something to trigger the docs changes check when a change has been made to `integrations/terraform/examples` ?